openapi.yaml

```diff
@@ -118,54 +161,68 @@ paths:
-        - $ref: '#/components/parameters/record-offset'        
       responses: &forecastSnapshotReadResponses
         "200": 
           description: >
             The requested operating forecast snapshot is returned.
           content:
-            application/vnd.trolie.forecast-limit-set.v1+json:
+            application/vnd.trolie.forecast-limit-set-slim.v1+json:
               schema:
                 $ref: '#/components/schemas/forecast-limit-set'
               example:
-                _links:
-                  self: 
-                    href: /limits/forecast-snapshot
-                total-count: 1
-                offset: 0
-                _embedded:
-                  forecasts:
-                  - _links:
-                      provenant-proposals:
-                      - href: /rating-proposals/forecasts/12345
-                      transmission-facility:
-                        href: /reference-data/transmission-facilities/line2
-                        name: line2
-                    periods:
-                    - period-start: "2023-07-12T16:00:00-07:00"
-                      updated-time: "2023-07-12T13:05:43.044267100-07:00"
-                      values: 
-                      - type: normal
-                        value:
-                          mva: 160
-                      - type: emergency
-                        value:
-                          mva: 165
-                      - type: loadshed
-                        value:
-                          mva: 170
+              - transmission-facility-id: line2
+                periods:
+                - period-start: "2023-07-12T16:00:00-07:00"
+                  updated-time: "2023-07-12T13:05:43.044267100-07:00"
+                  values: 
+                  - type: normal
+                    value:
+                      mva: 160
+                  - type: emergency
+                    value:
+                      mva: 165
+                  - type: loadshed
+                    value:
+                      mva: 170
+            application/vnd.trolie.forecast-limit-set-detailed.v1+json:
+              schema:
+                $ref: '#/components/schemas/forecast-limit-set-detailed'
+              example:
+              - transmission-facility-id: line2
+                periods:
+                - period-start: "2023-07-12T16:00:00-07:00"
+                  updated-time: "2023-07-12T13:05:43.044267100-07:00"
+                  provenant-proposals:
+                  - c386503e-ae8f-46e7-8fcf-bdee30f4f56b
+                  temporary-aar-exceptions:
+                  - 2d8c80e8-f533-4be9-85bf-f7f81eb73d67    
+                  limiting-segment: segmentX    
+                  override-reason: Any reason entered by the operator for an override.
+                  additional-data:
+                    vendor-specific-data: {}                         
+                  values: 
+                  - type: normal
```

Comment:
More of a breadcrumb for later, and I think you probably have it covered with this schema, but I think we'll need some Ratings-Provider-level indication of the duration in minutes of each of these types. This goes to that "we don't want to model" friction with prepositioning.

Maybe the right solution here is to think of the names, i.e., normal, emergency, load shed like we've been talking about for the facility names, but the **duration** is the really important thing. PJM has four tiers--two emergency ones, short- and long-term, and MISO uses only one of them since the other doesn't accommodate our emergency duration. The RC could validate that there's an acceptable rating for each of their tiers. This provides flexibility down the road if a given Ratings Provider operates with different tiers; our tariff requires they give us a normal/emergency/load shed, not that they don't have stuff in between.

I'm going to create an issue for this in TROLIE/spec

Reply:
Agreed.  While the names are probably important for interop, as they are what everybody understands, it's kind of an awkward link back to each grid operators' (both TOs and RCs) operations manuals.  Under the hood, it might be better to think of these as "sustained", "1 hour", and "15 minutes".  Or whatever they map to, I just made those up.